### PR TITLE
Display Cross-Domain confirmation only when Multistore Active

### PR DESCRIPTION
**Pull request by leemyongpakvn**
In `function treat` that process ConfigurationForm save at https://github.com/PrestaShop/ps_googleanalytics/blob/dev/classes/Form/ConfigurationForm.php#L205-L229
If `GA_CROSSDOMAIN_ENABLED` is not set, `Tools::getValue('GA_CROSSDOMAIN_ENABLED')` will return `False`, then `$gaCrossdomainEnabled` is assigned to `False`.
In turn `(null !== $gaCrossdomainEnabled)` is `True` because `(null !== False)` is `True`.
That's why ConfigurationForm Save display CrossDomain confirmation even when multistore is `Disabled`. Furthermore, CrossDomain confirmation is using UserIDTracking's Translation string, not its own.
ps_googleanalytics_multishop_enabled

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Solution: calculate and use `$is_multistore_active` in `function treat` like `function generate` did at line 49 and 169, add a new Translation string also. 
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes PrestaShop/PrestaShop/issues/28005
| How to test?      | After applying this PR: with multistore `Disabled` the wrong confirmation described in issue 28005 will disappear; with multistore `Enabled` and more than 1 shop the right confirmation will display like above screenshot.
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

### CODE REVIEW

**Comment by leemyongpakvn** [comment]
@makts As we discussed in the original PR 121: for single-store users, they won't see Cross Domain setting due to [Line 49](https://github.com/PrestaShop/ps_googleanalytics/pull/126/files#diff-5f981589f808927ab76592e5f9220ee730f4f05ecea0f9a10e0eef676546d5f2R49) and [Line 169-187](https://github.com/PrestaShop/ps_googleanalytics/pull/126/files#diff-5f981589f808927ab76592e5f9220ee730f4f05ecea0f9a10e0eef676546d5f2L169-L187), so this PR won't break the current logic.

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thank you, @leemyongpakvn - it is quite a significant issue. Nice catch.

**Review by HanaRebaiQA — COMMENTED** [review]
Hello @leemyongpakvn 

I have checked this PR when installing it on the final `8.0.0 version`. It seems that the issue is not fixed.
I always have the same error when the field `Google Analytics Tracking ID` is empty for the first time and even when putting a value and deleting it then.
I checked when multistore is disabled and enabled by adding different shops

image

https://user-images.githubusercontent.com/92912932/199281145-885858b3-7687-4d7c-a9db-aff3648c37ae.mp4



What do you think ?

Thank You!

**Comment by leemyongpakvn** [comment]
@HanaRebaiQA I can see you tested with **Single** store from 0:00 to 0:47 and **Multi** store from 0:48 to the end of the screencast, and do not see any issue. Can you let me know the exact time you took the screenshot (with 2 line of _'Settings for User ID updated successfully'_) from the screencast ?

**Comment by leemyongpakvn** [comment]
Hello @HanaRebaiQA I'm waiting for your answer :pray:

**Review by HanaRebaiQA — COMMENTED** [review]
Hello @leemyongpakvn 

Yes! Indeed when i saved the first time without putting any value, i have the double lines `Settings for User ID updated successfully` (See the mentioned **screen-shot** above)

But, when i have setted a value for the `Settings for User ID updated successfully` line i did not reproduce the double line.

If you want, to move again to` Waiting for QA` to check it again ?

Thank You

**Comment by leemyongpakvn** [comment]
@HanaRebaiQA I don't have write access right to can not change label myself. It looks like you forgot to apply [PR's Files changed](https://github.com/PrestaShop/ps_googleanalytics/pull/126/files) before testing.
@kpodemski Can you help in this case?

**Comment by kpodemski** [comment]
@leemyongpakvn maybe you need to consider joining us as a committer? :)

@HanaRebaiQA I don't see anything on the recording 🤔 I'm putting "Waiting for QA" again

**Comment by leemyongpakvn** [comment]
@kpodemski Maybe, we have just got 3 new committers last week - quite enough, so I will try next year ;)

**Comment by HanaRebaiQA** [comment]
Hello,

@kpodemski yes! the problem is displayed in the screenshot. But, we (QA team) can check it again.

image

Thank You!

**Review by hibatallahAouadni — COMMENTED** [review]
Hello @leemyongpakvn 

LGTM, QA :heavy_check_mark: 
But I have a question (it may seem irrelevant to your PR), shouldn't we show an error when we click save while this field is empty?

image

Waiting for your feedback.

Thanks!

**Comment by leemyongpakvn** [comment]
@hibatallahAouadni Yes, it is _expected_, because **Account ID** is set to _required_ and showed with a red asterisk. However, show a warning and/or force to input Account ID is out of scope of this PR.

**Comment by leemyongpakvn** [comment]
@hibatallahAouadni I'm waiting for your decision here :pray:

**Comment by kpodemski** [comment]
@leemyongpakvn I think we can agree that it's out of the scope of this PR

@hibatallahAouadni already validated the PR so I'm going to merge it, and we can improve the UX with a relevant warning in a different pull request :)

**Comment by kpodemski** [comment]
Thanks @leemyongpakvn :)

**Pull request merged**
